Update README.md (#321)
Layr-Labs/eigensdk-go: README.md

![Go Coverage](https://github.com/Layr-Labs/eigensdk-go/wiki/coverage.svg)
[![Go Report Card](https://goreportcard.com/badge/github.com/Layr-Labs/eigensdk-go)](https://goreportcard.com/report/github.com/Layr-Labs/eigensdk-go)

## EigenSDK

> This SDK provides a set of primitive Go modules for developing AVSs used in EigenLayer.

Reviewer: « in Eigenlayer » sounds a bit weird to me. Is there another formulation we could use instead?

Author: Agreed, changed to "on" vs "used in"

> This SDK provides a set of primitive Go modules for developing AVSs used in EigenLayer.

Author: ```suggestion
This SDK provides a set of primitive Go modules for developing AVSs on EigenLayer.
```
